### PR TITLE
[EOBS-2166] Refusal frequency for new NEWS task applies configurable data

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
- Moved anything concerning refusal in **nh_ews** up to **nh_eobs** as refusals are meant to first be introduced in that module.
- Moved logic for getting refusal frequency from the **nh.clinical.patient.observation.ews** model to the new **nh.clinical.frequencies.ews** model and refactored it.
- Updated tests to use configured values rather than hardcoded ones.
- Added new unit tests.

---

Before this pull request can be merged the following must be true.
- [ ] Unit tests pass (Travis integration).
- [ ] No code quality issues (Codacy integration).
- [ ] Approval from at least one developer and at least one tester.

If you are a BJSS contributor there are some additional conditions.
- [ ] All client module unit tests pass.
- [ ] The *Pull Request* field in the work tab of the JIRA issue is populated.
- [ ] The JIRA issue contains a description of the root cause and the solution in the comments.
- [ ] There are no bugs against the JIRA issues in the current sprint.

### CODE REVIEW

File: nh_eobs/tests/common/nh_clinical_test_utils.py [hunk 1]
```diff
@@ -9,6 +9,68 @@ class NhClinicalTestUtils(AbstractModel):
...
+    def complete_obs(self, obs_data, obs_activity_id=None, user_id=None):
+        """
+        Override to use API implementation as it is simpler.
+
+        :param obs_data:
+        :param obs_activity_id:
+        :param user_id:
+        :return:
+        """
+        api_pool = self.pool['nh.eobs.api']
```
Comment (Gimpneek) [inline nh_eobs/tests/common/nh_clinical_test_utils.py]: Is it possible to use `self.env` with nh.eobs.api so you can remove the `cr` and `uid` from the method calls?

File: nh_eobs/tests/nh_clinical_frequencies_ews/test_get_refusal_frequency.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        self.test_utils.admit_and_place_patient(create_placement=False)
+        self.test_utils.copy_instance_variables(self)
+
+        self.config_model.set_param('placement', 30)
+        self.config_model.set_param('transfer', 60)
+        self.config_model.set_param('obs_restart', 120)
+        self.config_model.set_param('no_risk', 240)
+
+    def call_test(self):
+        refused_obs_activity = \
```
Comment (Gimpneek) [inline nh_eobs/tests/nh_clinical_frequencies_ews/test_get_refusal_frequency.py]: Maybe add a docstring here?

Something like: `Refuse the open observation and return the new frequency in order to verify is correct`

File: nh_eobs/tests/nh_clinical_patient_observation_ews/test_update_next_obs_after_partial.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          1. Full obs.
+          2. Refused obs.
+          3. New obs scheduled but not yet completed.
+
+        The new obs at step 3 should have been scheduled using the same
+        frequency as the full obs before the refusal as that is the patients
+        most recent known clinical risk. The frequency is not altered by the
+        refusal in this case.
+
+        :return:
```
Comment (Gimpneek) [inline nh_eobs/tests/nh_clinical_patient_observation_ews/test_update_next_obs_after_partial.py]: Need to remove `:return:` as tests down return owt

File: nh_eobs/tests/common/nh_clinical_test_utils.py (hunk 1 above)
Comment (moderatemisbehaviour) [reply nh_eobs/tests/common/nh_clinical_test_utils.py]: Will do.

File: nh_eobs/tests/nh_clinical_frequencies_ews/test_get_refusal_frequency.py (hunk 2 above)
Comment (moderatemisbehaviour) [reply nh_eobs/tests/nh_clinical_frequencies_ews/test_get_refusal_frequency.py]: Yeah sure.

File: nh_eobs/tests/nh_clinical_patient_observation_ews/test_update_next_obs_after_partial.py (hunk 3 above)
Comment (moderatemisbehaviour) [reply nh_eobs/tests/nh_clinical_patient_observation_ews/test_update_next_obs_after_partial.py]: Cool thanks.

**Pull request merged**